common/persistence/persistence-tests/historyV2PersistenceTest.go

```diff
@@ -880,14 +880,15 @@ func (s *HistoryV2PersistenceSuite) append(ctx context.Context, branch []byte, e
 func (s *HistoryV2PersistenceSuite) fork(ctx context.Context, forkBranch []byte, forkNodeID int64) ([]byte, error) {
 
 	bi := []byte{}
-
+	domainName := s.DomainManager.GetName()
```

Comment:
nit: This value doesn't make sense here.
But this is only used in tests, so it's ok to leave domainName empty at this place.